content/applications/general/payment_acquirers/paypal.rst

```diff
@@ -18,29 +18,34 @@ Credentials tab
 
 Odoo needs your **API Credentials** to connect with your PayPal account, which comprise:
 
-- **Email ID**: your login email address in Paypal.
-- **Merchant ID**: the code of the merchant account used to identify your Paypal account.
-- **Use IPN**: either you want to use Instant Payment Notification. Already checked, you don't have
+- **Email**: your login email address in Paypal.
+- **Merchant Account ID**: the code of the merchant account used to identify your Paypal account.
+- **PDT Identity Token**: the key used to verify the authenticity of transactions.
+- **Use IPN**: whether you want to use Instant Payment Notification. Already checked, you don't have
   to change it.
 
-You can copy your credentials from your Paypal account, and paste them in the related fields under
+You can copy your credentials from your Paypal account and paste them into the related fields under
 the **Credentials** tab.
 
-To retrieve them, log into your Paypal account, and go to :menuselection:`Your account menu -->
-Account settings --> Business Profile --> Business Information`
+To retrieve the **Merchant Account ID**, log into your Paypal account and go to
+:menuselection:`account menu --> Account Settings --> Business information`.
```

Comment:
I'm not sure of the best way to formulate this... Originally, it was "go to **Your account menu** -> ...", then I changed it to "go to your **account menu** -> ...". In practice, the Account Settings button is in the "account menu" which is named after your PayPal user. :point_down: 

![image](https://user-images.githubusercontent.com/15896813/151170660-dd39b1c1-cefa-4ec3-9292-0d092388013c.png)